diff --git a/modules/wingsuit_ui_patterns/src/Plugin/Deriver/LibraryDeriver.php b/modules/wingsuit_ui_patterns/src/Plugin/Deriver/LibraryDeriver.php
index d5f4da4..a58401f 100644
--- a/modules/wingsuit_ui_patterns/src/Plugin/Deriver/LibraryDeriver.php
+++ b/modules/wingsuit_ui_patterns/src/Plugin/Deriver/LibraryDeriver.php
@@ -212,7 +212,11 @@ class LibraryDeriver extends AbstractYamlPatternsDeriver {
    */
   protected function getDirectories() {
     $dist_path = $this->config->get('dist_path');
+    // Default wingsuit provider.
     $directories['wingsuit_companion'] = realpath($dist_path);
+    // Add default theme as a provider.
+    $directories[$this->themeHandler->getDefault()] = realpath($dist_path);
+
     return $directories;
   }
 
